src/toolbars/TimeSignatureToolBar.cpp

```diff
@@ -104,11 +104,16 @@ void TimeSignatureToolBar::Create(wxWindow* parent)
 
 void TimeSignatureToolBar::Populate()
 {
+#ifndef __WXGTK__
    const auto tempoSigSize = wxSize(60, -1);
    auto timeSigSize = wxSize(50, -1);
+#else
+   const auto tempoSigSize = wxSize(60, -1);
```

Comment:
`tempoSigSize` values are same in both branches, is that correct?

Reply:
Yep